eng/Versions.props

```diff
@@ -17,7 +17,7 @@
     <MicrosoftNETCoreILAsmVersion>6.0.0-preview.6.21352.12</MicrosoftNETCoreILAsmVersion>
     <MicrosoftNETCoreILDAsmVersion>6.0.0-preview.6.21352.12</MicrosoftNETCoreILDAsmVersion>
```

Comment:
@ViktorHofer this version is out of sync with the one in Version.Details.xml and looks manually updated, should we remove it there?

Reply:
Isn't that version used by the repo build?

Reply:
We should probably synchronize them and bring them up-to-date

Reply:
OK this was already the case before the backflow changes. @MichaelSimons should those version get updated to something newer? A version closer to what the product uses with the PSB? Also, should there be toolset sub for it?